src/include/sof/schedule/edf_schedule.h

```diff
@@ -35,4 +35,6 @@ struct edf_task_pdata {
 	uint64_t deadline;
 };
 
+int edf_scheduler_init(void);
```

Comment:
Since this is an external API we should prefix it with scheduler_, this way API will have a more searchable and meaningful API space. e.g. scheduler_init_edf()